case_studies/redshift_estimation/catalog.py

```diff
@@ -0,0 +1,50 @@
+import torch
+from einops import rearrange, reduce, repeat
+
+from bliss.catalog import FullCatalog, TileCatalog
+
+
+class RedshiftTileCatalog(TileCatalog):
```

Comment:
We should be able to delete these redshift-specific catalogs by using TileCatalog and FullCatalog instead to store redshift.

Reply:
I think this is undergraduate wrote and not needed to be merged to master branch. So I deleted whole redshift_estimation dir now.